computation/src/main/java/com/powsybl/computation/DefaultExecutionReport.java

```diff
@@ -22,29 +27,22 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(DefaultExecutionReport.class);
 
-    private static final ExecutionReport OK = new DefaultExecutionReport();
+    private final Path workingDirectory;
 
     private final List<ExecutionError> errors;
 
-    /**
-     * An execution report with no execution error.
-     * @return an execution report with no execution error.
-     */
-    public static ExecutionReport ok() {
```

Comment:
Just for me, when did we use this method? (Don't forget the Breaking change label if we delete it)

Reply:
This was used only in the unit test, but it's really useful to return always the same instance.